integration/maptest/map.go

```diff
@@ -163,24 +168,28 @@ func RunMapRevisionZero(ctx context.Context, t *testing.T, tadmin trillian.Trill
 		},
 	} {
 		for _, hashStrategy := range tc.hashStrategy {
```

Comment:
This pattern of testing is slightly easier if we put it in the following form: 
```
t.Run(tc.desc, func (t testing.T) {
    ...
    if err != nil {
         t.Fatalf()
    }
})
```

This has the following nice benefits: 
- Individual test cases can be isolated when running with `go test ./integration/maptest -run RunMapRevisionZero/Desc`
- Test descriptions don't need to be included in every error message
- `t.Fatalf` and `return` can be used without stopping tests later on.